guide/src/links.md

```diff
@@ -25,7 +25,7 @@
     DITTO LINKS
 )
 [Ditto]: https://www.ditto.live/about/company
-[Ditto-logo]: /safer_ffi/assets/ditto-small.png
+[Ditto-logo]: assets/ditto-logo.png
```

Comment:
Markdown / mdbook does not seem to offer a way to resize the images, so we will either need to generate straight html code and css to do it for us, or provide lower resolution images 🤔